lib/adviser_list_csv.rb

```diff
@@ -5,11 +5,33 @@ def initialize(advisers)
 
   def to_csv(_ = {})
     CSV.generate do |csv|
-      csv << ['Ref. Number', 'Name', 'Firm']
-
+      csv << headings
       @advisers.each do |adviser|
-        csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+        csv << row(adviser)
       end
     end
   end
+
+  private
+
+  def headings
+    ['Ref. Number', 'Name', 'Firm'] + all_certifications
+  end
+
+  def all_certifications
+    @all_certifications ||= Accreditation.all.pluck(:name) + Qualification.all.pluck(:name)
+  end
+
+  def certified(adviser)
+    adviser_certifications = adviser.accreditations.pluck(:name) + adviser.qualifications.pluck(:name)
```

Comment:
please address the line length for this line and the one below it

Reply:
Hi @margOnline. I'm just going through and sorting out the line lengths now.
Metrics/LineLength seems to be set to 120 in the RAD rubocop file, but in the example from the PR Approval doc it's 80. Shall I update this in RAD or would you prefer I just run against the example rubocop file instead?

Reply:
Good question. I have a feeling that we haven't updated it in the RAD repo because it would throw too many errors. 

We are try to update to the new standards with every file we touch. As per the previous engagement with Cultivate, we ask that you all do the same. Appreciate that its tedious but its all part of the job. Perhaps this is something we can discuss when you are in London later this week?

If you dont mind updating the RAD rubocop file and running it just to see what it throws, that would be great.